TalismanSociety/talisman-web
# Feat: Manta staking

# Description

- Add Manta to staking dashboard
- Add Manta and vManta to balances custom tokens
- Fix "Staking % bar" 

## Type of change


### 🧪 **How to test:**
-  Get some Manta tokens on Manta Pacific
- Get some ETH for gas fees on Manta pacific
- Visit `staking/providers`
- Click on Stake

Expected:
- Manta and vManta balances should be displayed
- vManta stake balance should be displayed on `staking/positions`
- Staking and Unstaking should be possible

### 🗒️ **Notes:**

- Staking may take up to 15 mins to get the tokens after the tx has been mined
- Unstaking usually takes up to 15mins, but may be subjected to 0~7 Days unstaking period. If that is the case tokens will be displayed on the UI as locked in 'staking/positions', no claiming required when the lock period is up.


### 🖼️ **Screenshots:**


https://github.com/user-attachments/assets/38f910a8-319c-434e-87d2-e373b99a21bd
